src/main/java/com/ithaca/user/User.java

```diff
@@ -13,8 +14,10 @@
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
+    @NotNull
     private String name;
 
+    @NotNull
```

Comment:
@NotNull ensures that when the POJO is going to be stored, it has to have those two values. Basically we cannot have a row in the database for a User have empty columns for name and password.